content/docs/setup/kubernetes/multicluster-install/zvpn/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Using Istio Gateway
+description: Connect multiple clusters using Istio Gateway to reach remote pods.
+weight: 2
+keywords: [kubernetes,multicluster,gateway]
+---
+
+Instructions for spanning an Istio mesh across multiple clusters when pods
+in each cluster can only talk to remote gateway IPs.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters with **1.9 or newer**.
+
+* The ability to deploy the [Istio control plane](/docs/setup/kubernetes/quick-start/)
+on **each** Kubernetes cluster.
+
+* The gateway IP (load balancer IP) in each cluster must be accessible
+from every other cluster.
+
+* **Root CA**: To enable mTLS communication across clusters, each cluster's
+  Citadel must be configured with an intermediate CA credentials generated
+  by a shared root CA.
+
+## Overview
+
+In this mode, each cluster has an **identical** Istio control plane
+installation. Cross-cluster communication occurs over Istio Gateways
+of the respective clusters. 
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-zvpn.svg"
+    caption="Spanning a single mesh across multiple clusters using Istio Gateways"
+    >}}
+
+This guide describes steps to install a multicluster Istio topology using
+the manifests and Helm charts provided as part of Istio install.
+
+## Deploy Istio control plane in each cluster
+
+Install the Istio control plane using the Helm command below:
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --set multiCluster.connectUsingGateway=true \
+    --set security.selfSigned=false > $HOME/istio.yaml
+$ kubectl create namespace istio-system
+$ kubectl create secret generic cacerts -n istio-system --from-file=path/to/intermediate/ca-cert.pem \
+    --from-file=path/to/intermediate/ca-key.pem --from-file=path/to/intermediate/root-cert.pem \
+    --from-file=path/to/intermediate/cert-chain.pem
+$ kubectl apply -f $HOME/istio.yaml
+{{< /text >}}
+
+For further details and customization options, refer to the [Installation
+with Helm](/docs/setup/kubernetes/helm-install/) page.
+
+Note that the installation steps above assume that you have generated an
```

Comment:
Recommend replace `Note that the` with `> The`.  Do we have any already-available documentation on generating intermediate certs?  If so a link would be great.